sql/src/main/java/io/crate/operation/collect/stats/TimeExpiring.java

```diff
@@ -34,16 +34,15 @@
 class TimeExpiring {
 
     /**
-     * clean interval in seconds
+     * clean interval in milliseconds
      */
-    private static final long QUEUE_CLEAN_INTERVAL = 5L;
+    private static final long QUEUE_CLEAN_INTERVAL = 5000L;
 
     private static TimeExpiring INSTANCE = new TimeExpiring(QUEUE_CLEAN_INTERVAL, 0L);
     private final long interval;
     private final long delay;
 
-    @VisibleForTesting
-    TimeExpiring(long interval, long delay) {
+    public TimeExpiring(long interval, long delay) {
```

Comment:
keep `TimeExpiring(long interval, long delay)` package private and add a new constructor `TimeExpiring(long interval)`